DirectEmployers/MyJobs
# [PD-3872] Use credentials to connect to ElasticSearch

## What's this PR do?
Updates all the settings for ElasticSearch to use credentials to connect.

## Where should the reviewer start?
The settings files.
There are only two ES clusters to chose from - test (covers QC and staging) and production.

## How should this be manually tested?
Connect to ES and add/search for/remove some jobs.
Automated tests should cover these actions relatively well.

## Any background context you want to provide?
This is being done as an alternative to whitelisting, which is how I handle permissions currently. Unfortunately, whitelisting isn't a good long-term solution. 
I am ashamed of how long this took for me to figure out.

## PR Checklist

### Separation of Concerns
